game.js

```diff
@@ -0,0 +1,48 @@
+const COMP_WINS = 0;
+const USER_WINS = 1;
+
+const computerSelection = computerPlay();
```

Comment:
It seems like you have to reload the page to play a round. I suggest to call `computerPlay` directly in the click handler. That way you play a round any time you click a button.